### PR TITLE
Curriculum update

### PR DESCRIPTION
**Pull request by Del-sama**

### CODE REVIEW

File: other important stuff/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+* [Cracking the coding interview](https://books.google.co.uk/books/about/Cracking_the_Coding_Interview.html?id=anhAXwAACAAJ&hl=en)
+
+* [How I landed my first job as a software developer...](https://www.freecodecamp.org/news/how-i-set-myself-up-to-land-my-first-job-as-a-self-taught-developer-43d326ea6b67/)
+
+* [Interview tips from Google software engineers](https://www.youtube.com/watch?v=XOtrOSatBoY&feature=youtu.be)
+
+* [Shell scripting](https://www.shellscript.sh/interactive.html)
+
+### TBD
```
Comment (kosyfrances) [inline other important stuff/README.md]: How about renaming the directory to `other` or `random`?

File: python/README.md [hunk 2]
```diff
@@ -54,10 +69,18 @@ The program will first randomly generate a number unknown to the user. The user
...
 
 * [PEP 8 - Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/)
 
 * [CS50's Introduction to Computer Science](https://www.edx.org/course/cs50s-introduction-to-computer-science)
+
+* [Python for Non-Programmers](https://wiki.python.org/moin/BeginnersGuide/NonProgrammers)
+
+* [How to think like a Computer Scientist](http://interactivepython.org/runestone/static/thinkcspy/index.html)
+
+* [Real Pyton](https://realpython.com/)
```
Comment (kosyfrances) [inline python/README.md]: s/Pyton/Python

**Review by kosyfrances — CHANGES_REQUESTED** [review]
Thanks @Del-sama for this. Just a few comments to fix and we are good to 🚢

File: python/README.md (hunk 2 above)
Comment (Del-sama) [reply python/README.md]: I don't get this

Comment (kosyfrances) [reply python/README.md]: I meant, `Real Pyton` should be `Real Python`

Comment (Del-sama) [reply python/README.md]: Fixed

**Review by kosyfrances — CHANGES_REQUESTED** [review]
Thanks for the changes @Del-sama. One more thing left. Please can you remove the .DS_Store file you mistakenly added?

**Review by kosyfrances — APPROVED** [review]

**Pull request merged**